<launch>
    <!--############### Node Configurations ##################-->
    <arg name="output" default="screen"/>
    <arg name="enabled" default="true"/>
    <!-- must know the camera frame rate and the max camera free running
         frame rate to limit max exposure time! --> 
    <arg name="frame_rate"/>
    <arg name="max_fps_free" default="200"/>
    <!--- start the nodelet manager-->
    <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" output="screen"/>
    
    <!-- nodelet -->
    <node pkg="nodelet" type="nodelet" name="exposure_control"
	  args="load exposure_control/ExposureControlNodelet nodelet_manager"
	  output="$(arg output)">
        <param name="fps" type="double" value="$(arg frame_rate)"/>
	<param name="max_fps_free" type="double" value="$(arg max_fps_free)"/>
	<remap from="exposure_control/image" to="cam_test/cam0/image_raw"/>
	<remap from="exposure_control/current_exposure" to="/cam_test/current_exposure"/>
    </node>
</launch>
